webpack/ForemanInventoryUpload/Components/InventorySettings/AdvancedSetting/AdvancedSettingActions.js

```diff
@@ -0,0 +1,7 @@
+import { setSetting } from '../InventorySettingsActions';
+
+export const handleToggle = (name, current) =>
+  setSetting({
+    setting: name,
+    value: !current,
+  });
```

Comment:
can we get just the name of the setting here,
and select its current value from the store?

e.g:

```suggestion
export const handleToggle = (name) => (dispatch, store) => {
  dispatch(setSetting({
    setting: name,
    value: !selectSettingByName(store.getState(), name),
  }))
  }
```

Reply:
I think thats too complicated, I can change the call of the function to use `!settingValue` instead of `settingValue`
``` js

 const settingValue = useSelector(store => selectSettings(store)[setting]);
 const onToggle = () =>
    dispatch(handleToggle(settingsDict[setting].name, !settingValue));
```